NuGet.Config

```diff
@@ -6,6 +6,8 @@
     <add key="dotnet-core" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
     <add key="cli-deps" value="https://dotnet.myget.org/F/cli-deps/api/v3/index.json" />
     <add key="xunit" value="https://www.myget.org/F/xunit/api/v3/index.json" />
-    <add key="api.nuget.org" value="https://api.nuget.org/v3/index.json" />
+    <!-- Temporary -->
```

Comment:
We will. I am talking to the roslyn folks to add the webhook we need. If that does not happen. I will manually push the packages. I won't check it in with this feed in there, but I wanted feedback on the rest of the changes.